.circleci/config.yml

```diff
@@ -0,0 +1,70 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+version: 2.1
+
+commands:
+  build-common:
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - maven-repo-v1-{{ .Branch }}-{{ checksum "pom.xml" }}
+            - maven-repo-v1-{{ .Branch }}-
+            - maven-repo-v1-
+      - save_cache:
+          paths:
+            - ~/.m2
+          key: maven-repo-v1-{{ .Branch }}-{{ checksum "pom.xml" }}
+
+      - run:
+          name: Install build tools
+          command: |
+            sudo apt-get update -y
+            DEBIAN_FRONTEND=noninteractive \
+            DEBCONF_NONINTERACTIVE_SEEN=true \
+            TZ=UTC \
+            sudo apt-get install -y --no-install-recommends openjdk-8-jdk maven
+
+      - run:
+          name: Build
+          command: |
+            export JAVA_HOME="/usr/lib/jvm/java-8-openjdk-arm64/"
+            export PATH="$JAVA_HOME/bin:$PATH"
+            java -version
+            mvn -version
+            mvn clean verify -Pspark-3.0 -Pspark-scala-2.12 -DskipTests --also-make-dependents --batch-mode --no-transfer-progress
+
+jobs:
+  build-arm64:
+    machine:
+      image: ubuntu-2004:202101-01
```

Comment:
Is it necessary to pass this specific image version or can we use something like ubuntu-2004:latest?

Reply:
At the moment there are only two supported versions of the Ubuntu ARM64 images - https://circleci.com/docs/2.0/arm-resources/#overview.
```
Which are available under these images:

    ubuntu-2004:202101-01 - most recent, recommended for all users
    ubuntu-2004:202011-01 - deprecated as of Feb 3, 2021
```